### ISSUE TITLE
AutoCompleteInput non-case sensitive

### ISSUE THREAD

**Issue by ahuang11** [holoviz/panel#1430]
https://panel.holoviz.org/reference/widgets/AutocompleteInput.html

```
import panel as pn
pn.extension()
pn.widgets.AutocompleteInput(options=['test', 'Test2'])
```
![image](https://user-images.githubusercontent.com/15331990/85189341-d99ab780-b273-11ea-9003-112a6c9dea6c.png)

Can there be an option to ignore case sensitivity?

**Comment by philippjfr** [comment]
This was only just merged (https://github.com/bokeh/bokeh/pull/10099) so exposing that property will have to wait for a Bokeh 2.2 release.

**Comment by dwsideriusNIST** [comment]
Fix proposed in PR #1548
